src/webvr-polyfill.js

```diff
@@ -95,6 +95,18 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
 
   // Provide the VRDisplay object.
   window.VRDisplay = VRDisplay;
+
+  // Provide navigator.vrEnabled.
+  var that = this;
```

Comment:
nit: `self` is used [elsewhere](https://github.com/brianchirls/webvr-polyfill/blob/28d66968/src/webvr-polyfill.js#L135)